src/components/Modal/index.jsx

```diff
@@ -198,7 +198,7 @@ export default class Modal extends React.Component {
     }
 
     return (
-      <Portal closeOnEsc isOpened onClose={this.cancel}>
```

Comment:
@masylum @andresgutgon found a solution. 

`react-portal` passes a `closePortal` method to its immediate child. Wrapping modal content inside a `Content` component and calling `closePortal` instead of `cancel` does the trick 👍